libpod/volume.go

```diff
@@ -114,11 +122,17 @@ func (v *Volume) Anonymous() bool {
 
 // UID returns the UID the volume will be created as.
 func (v *Volume) UID() int {
+	if v.state.UIDChowned > 0 {
```

Comment:
These need to do a v.lock() / v.update() first - v.state is not populated until the first v.update happens

Reply:
We probably also need an unlocked version to use inside of volume_inspect below

Reply:
thanks, fixed in the new version